pkg/collector/corechecks/servicediscovery/module/impl_linux.go

```diff
@@ -491,6 +500,9 @@ func (s *discovery) getService(context parsingContext, pid int32) *model.Service
 	if name == "" {
 		name = info.generatedName
 	}
+	if s.shouldIgnoreService(name, proc) {
+		return nil
+	}
```

Comment:
why don't you add the pid to the ignore list?

Reply:
I've changed this code and separated ignored service and addition of ignored pid